addon/utils/is-fastboot.js

```diff
@@ -10,8 +10,18 @@ import ApplicationInstance from '@ember/application/instance';
  * @return {ComputedProperty<boolean>}
  */
 export default function isFastBootCPM() {
-  return computed(function() {
-    return isFastBoot(getOwner(this));
+  return computed({
+    get() {
+      if (this.hasOwnProperty('isFastBootOverride')) {
+        return this.isFastBootOverride;
+      } else {
+        return isFastBoot(getOwner(this));
+      }
+    },
+    set(key, value) {
+      this.isFastBootOverride = value;
+      return value;
+    }
```

Comment:
I don't understand this change. Why would something call `this.set('isFastboot`, true/false)`? Seems to me that this kind of thing won't change during the runtime of the app 🤔